src/mac.js

```diff
@@ -212,11 +222,15 @@ class MacApp extends App {
     }
 
     if (this.appVersion) {
-      this.appPlist.CFBundleShortVersionString = this.appPlist.CFBundleVersion = '' + this.appVersion
+      for (const plistKey of plistsToUpdate) {
+        this[plistKey].CFBundleShortVersionString = this[plistKey].CFBundleVersion = '' + this.appVersion
```

Comment:
`'' + this.appVersion` could be moved outside of the loop to ensure it's only built once. Same for buildVersion below